torch/_inductor/codegen/cpp.py

```diff
@@ -1730,7 +1730,10 @@ def codegen_loops(self, code, worksharing):
 
     @property
     def assert_function(self) -> str:
-        return "TORCH_CHECK"
+        if V.graph.aot_mode:
+            return "AOTI_CHECK"
+        else:
+            return "TORCH_CHECK"
```

Comment:
JIT Inductor will also face the compilation time challenge sooner or later, so it's worth to unify the behavior here.

Reply:
So, we can just remove the ```if V.graph.aot_mode``` condition? Thanks.